src/app/service/data-service.service.ts

```diff
@@ -0,0 +1,68 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DataService {
+  public data = [{
+    first: [
+      {
+        list: 'first',
+        id: 1,
+        title: 'First list title item',
+        description: 'Example description for first list title item',
+        completed: false,
+      },
+    ],
+    second: [
+      {
+        list: 'second',
+        id: 2,
+        title: 'Second list title item',
+        description: 'Example description for second list title item',
+        completed: false,
+      },
+    ],
+    third: [
+      {
+        list: 'third',
+        id: 3,
+        title: 'Third list title item',
+        description: 'Example description for third list title item',
+        completed: false,
+      },
+    ],
+  }];
+
+  loadAll() {
+    return this.data;
+  }
+  // Add an item in a specific list
+  addItem(list, item) {
+    if (list in this.data[0]) {
+      this.data[0][list].push(item);
+    } else {
+      this.addList(list, item);
+    }
+  }
+  // Add a list whit an item inside, or an empty list
+  addList(list, item) {
```

Comment:
What are the types of list and item?

You can make this function variadic in order to insert any number of elements
`addList(list: ListType, ...items: ItemType[])`